exchanges/binance/binance.go

```diff
@@ -1267,3 +1299,475 @@ func (b *Binance) FetchSpotExchangeLimits(ctx context.Context) ([]order.MinMaxLe
+	}
+	params.Set("collateralCoin", collateralCoin.String())
+	if collateralAmount != 0 {
+		params.Set("collateralAmount", strconv.FormatFloat(collateralAmount, 'f', -1, 64))
+	}
+	params.Set("loanTerm", strconv.FormatInt(loanTerm, 10))
+
+	var resp []CryptoLoanBorrow
+	return resp, b.SendAuthHTTPRequest(ctx, exchange.RestSpotSupplementary, http.MethodPost, loanBorrow, params, spotDefaultRate, &resp)
+}
+
+// CryptoLoanBorrowHistory gets loan borrow history
+func (b *Binance) CryptoLoanBorrowHistory(ctx context.Context, orderID int64, loanCoin, collateralCoin currency.Code, startTime, endTime time.Time, current, limit int64) (*LoanBorrowHistory, error) {
+	params := url.Values{}
+	if orderID != 0 {
+		params.Set("orderId", strconv.FormatInt(orderID, 10))
+	}
+	if !loanCoin.IsEmpty() {
+		params.Set("loanCoin", loanCoin.String())
+	}
+	if !collateralCoin.IsEmpty() {
+		params.Set("collateralCoin", collateralCoin.String())
+	}
+	if !startTime.IsZero() {
+		params.Set("startTime", strconv.FormatInt(startTime.UnixMilli(), 10))
+	}
+	if !endTime.IsZero() {
+		params.Set("endTime", strconv.FormatInt(endTime.UnixMilli(), 10))
+	}
+	if current != 0 {
+		params.Set("current", strconv.FormatInt(current, 10))
+	}
+	if limit != 0 {
+		params.Set("limit", strconv.FormatInt(limit, 10))
+	}
+
+	var resp LoanBorrowHistory
+	return &resp, b.SendAuthHTTPRequest(ctx, exchange.RestSpotSupplementary, http.MethodGet, loanBorrowHistory, params, spotDefaultRate, &resp)
+}
+
+// CryptoLoanOngoingOrders obtains ongoing loan orders
+func (b *Binance) CryptoLoanOngoingOrders(ctx context.Context, orderID int64, loanCoin, collateralCoin currency.Code, current, limit int64) (*CryptoLoanOngoingOrder, error) {
+	params := url.Values{}
+	if orderID != 0 {
+		params.Set("orderId", strconv.FormatInt(orderID, 10))
+	}
+	if !loanCoin.IsEmpty() {
+		params.Set("loanCoin", loanCoin.String())
+	}
+	if !collateralCoin.IsEmpty() {
+		params.Set("collateralCoin", collateralCoin.String())
+	}
+	if current != 0 {
+		params.Set("current", strconv.FormatInt(current, 10))
+	}
+	if limit != 0 {
+		params.Set("limit", strconv.FormatInt(limit, 10))
+	}
+
+	var resp CryptoLoanOngoingOrder
+	return &resp, b.SendAuthHTTPRequest(ctx, exchange.RestSpotSupplementary, http.MethodGet, loanOngoingOrders, params, spotDefaultRate, &resp)
+}
+
+// CryptoLoanRepay repays a crypto loan
+func (b *Binance) CryptoLoanRepay(ctx context.Context, orderID int64, amount float64, repayType int64, collateralReturn bool) ([]CryptoLoanRepay, error) {
+	if orderID <= 0 {
+		return nil, errOrderIDMustBeSet
+	}
+	if amount <= 0 {
+		return nil, errAmountMustBeSet
+	}
+
+	params := url.Values{}
+	params.Set("orderId", strconv.FormatInt(orderID, 10))
+	params.Set("amount", strconv.FormatFloat(amount, 'f', -1, 64))
+	if repayType != 0 {
+		params.Set("type", strconv.FormatInt(repayType, 10))
+	}
+	if collateralReturn {
+		params.Set("collateralReturn", "true")
+	}
```

Comment:
Same with here, it defaults to true
https://binance-docs.github.io/apidocs/spot/en/#repay-crypto-loan-repay-trade